composites/Plugin/Shared/components/KeywordInput.js

```diff
@@ -112,14 +115,20 @@ class KeywordInput extends React.Component {
 				{ showLabel && <KeywordFieldLabel htmlFor={ id }>
 					{ label }
 				</KeywordFieldLabel> }
-				<KeywordField
-					aria-label={ showLabel ? null : label }
-					type="text"
-					id={ id }
-					className={ showErrorMessage ? "hasError" : null }
-					onChange={ this.handleChange }
-					value={ keyword }
-				/>
+				<YoastInputButtonContainer>
+					<KeywordField
+						aria-label={ showLabel ? null : label }
+						type="text"
+						id={ id }
+						className={ showErrorMessage ? "hasError" : null }
+						onChange={ this.handleChange }
+						value={ keyword }
+					/>
+					<YoastInputClearButton
+						onClick={ this.props.onRemoveKeyword }
```

Comment:
Why isn't this being assigned the same way as the other props are? See first line of this method.